activerecord/test/cases/schema_dumper_test.rb

```diff
@@ -192,7 +174,7 @@ def test_schema_dumps_index_columns_in_right_order
 
   def test_schema_dumps_partial_indices
     index_definition = dump_table_schema("companies").split(/\n/).grep(/t\.index.*company_partial_index/).first.strip
-    if current_adapter?(:PostgreSQLAdapter, :SQLite3Adapter) && ActiveRecord::Base.connection.supports_partial_index?
+    if current_adapter?(:PostgreSQLAdapter, :SQLite3Adapter)
```

Comment:
I'm not sure whether this should be using an adapter type check, or the `supports_partial_index?` condition (as we do for index sort order below). Agree it definitely shouldn't be both.